File: db4oj.tests/src/com/db4o/db4ounit/jre12/handlers/AllTests.java
/* Copyright (C) 2006 db4objects Inc. http://www.db4o.com */

package com.db4o.db4ounit.jre12.handlers;

import db4ounit.extensions.Db4oTestSuite;

/**
 * @decaf.ignore.jdk11
 */
public class AllTests extends Db4oTestSuite {

	protected Class[] testCases() {
		return new Class[] { 
				TypeHandlerConfigurationTestCase.class, 
		};
	}

}

File: db4o.net/Db4objects.Db4o.Tests/Db4objects.Db4o.Tests/Common/Staging/AllTests.cs
/* Copyright (C) 2004 - 2008  Versant Inc.  http://www.db4o.com */

using System;
using Db4oUnit.Extensions;
using Db4objects.Db4o.Tests.Common.Staging;

namespace Db4objects.Db4o.Tests.Common.Staging
{
	public class AllTests : ComposibleTestSuite
	{
		public static void Main(string[] args)
		{
			new Db4objects.Db4o.Tests.Common.Staging.AllTests().RunSolo();
		}

		protected override Type[] TestCases()
		{
			return ComposeTests(new Type[] { typeof(ActivateDepthTestCase), typeof(InterfaceQueryTestCase
				), typeof(LazyQueryDeleteTestCase), typeof(SODAClassTypeDescend), typeof(StoredClassUnknownClassQueryTestCase
				) });
		}

		// COR-1131
		#if !SILVERLIGHT
		protected override Type[] ComposeWith()
		{
			return new Type[] { typeof(ClientServerPingTestCase), typeof(DeepPrefetchingCacheConcurrencyTestCase
				), typeof(PingTestCase) };
		}
		#endif // !SILVERLIGHT
		// COR-1762
	}
}
